packages/datetime/src/_timepicker.scss

```diff
@@ -109,3 +109,13 @@ $timepicker-control-width: $pt-grid-size * 3.3 !default;
     color: $pt-dark-text-color;
   }
 }
+
+.#{$ns}-datepicker .#{$ns}-timepicker {
+  margin-bottom: $datepicker-padding * 2;
+  text-align: center;
+
+  // adjust margin when actions bar is hidden
+  &:last-child {
+    margin-bottom: $datepicker-padding;
+  }
+}
```

Comment:
I would consider adding `5px` of margin above the time picker. Feels a _little_ cramped right now, and with `5px` more of separation, a selected date on the bottom row yields a symmetric `10px` of white space both above and below the time picker.

![2018-08-21 09 42 33](https://user-images.githubusercontent.com/443450/44416000-a7f52000-a526-11e8-8bce-b350b834216c.gif)

Reply:
done